### DISCUSSION TITLE
Suggest Install documentation include UEFI requirement for Centos 7 in Security Onion iso

### DISCUSSION THREAD

**Discussion by mikeInCalgary** [Security-Onion-Solutions/securityonion#5432] [Unsupported Versions] [answered]
Tried installing securityonion-2.3.70-WAZUH.iso on one of my HP DL-380 Gen 6 m/c's. Home lab is made up of recycled servers from the ERA (Electronic Recyling Agency). HP startup comes back with message that UEFI is not acceptable. Recognize that the Gen 6 is a bit dated and apparently HP has only relatively recently added the UEFI BIOS option in Gen 9.

It's not that hard to work around - just install a hypervisor (eg. VMware, ProxMox or Virtual Box). Then install on top of that. Ubuntu 18.04 works fine with the legacy BIOS; however, this is not the base OS for the all-in-one SO-2 release.

### Discussed in https://github.com/Security-Onion-Solutions/securityonion/discussions/2959

<div type='discussions-op-text'>

<sup>Originally posted by **negus6** February 12, 2021</sup>
Booting to CentosOS Linux 7 Core is successful, but the installation hangs at starting installer. The server was running SO 16.04 previously. As a troubleshooting measure I deleted all the partitions from the disk using fdisk to blow away the old installation... then tried installing SO 2. I also tried booting UEFE and Legacy using both DVD and USB installation media (verified iso). Still no luck.</div>

**Comment by dougburks** [comment, marked as answer]
Our ISO image does indeed work fine on most BIOS and UEFI implementations. Not sure why it's not working on your Gen 6 hardware, but as you mentioned it is an older generation so that's a potential clue. In addition to the Ubuntu 18.04 option, you could also try a standard CentOS 7 installation and then install our components on top of that manually.
